tests/suites/test_suite_psa_crypto.data

```diff
@@ -1966,10 +1966,6 @@ PSA key agreement: ECDH SECP256R1 (RFC 5903), raw: read 32 (full)
 depends_on:MBEDTLS_PK_PARSE_C:MBEDTLS_ECP_C:MBEDTLS_ECP_DP_SECP256R1_ENABLED:MBEDTLS_ECDH_C
 key_agreement_output:PSA_ALG_ECDH:PSA_KEY_TYPE_ECC_KEYPAIR(PSA_ECC_CURVE_SECP256R1):"c88f01f510d9ac3f70a292daa2316de544e9aab8afe84049c62a9c57862d1433":"04d12dfb5289c8d4f81208b70270398c342296970a0bccb74c736fc7554494bf6356fbf3ca366cc23e8157854c13c58d6aac23f046ada30f8353e74f33039872ab":"d6840f6b42f6edafd13116e0e12565202fef8e9ece7dce03812464d04b9442de":""
 
-PSA key agreement: ECDH SECP256R1 with ECDH-only public key
-depends_on:MBEDTLS_PK_PARSE_C:MBEDTLS_ECP_C:MBEDTLS_ECP_DP_SECP256R1_ENABLED:MBEDTLS_ECDH_C
-key_agreement_output:PSA_ALG_ECDH:PSA_KEY_TYPE_ECC_KEYPAIR(PSA_ECC_CURVE_SECP256R1):"c88f01f510d9ac3f70a292daa2316de544e9aab8afe84049c62a9c57862d1433":"04d12dfb5289c8d4f81208b70270398c342296970a0bccb74c736fc7554494bf6356fbf3ca366cc23e8157854c13c58d6aac23f046ada30f8353e74f33039872ab":"d6840f6b42f6edafd13116e0e12565202fef8e9ece7dce03812464d04b9442de":""
-
```

Comment:
We should make the commit "Remove obsolete test case "ECDH-only public key" in the main development branch. Should be easy to do even today (no other dependencies). I don't see a reason to make this commit only in the API branch, or first in the API branch.

What did you have in mind for syncing the API and development branch in the future?

Reply:
I'm not going to redo the merge commit without a very good reason. Git will do the right thing if we make the same commit on both sides.

Ideally we wouldn't have separate branches for the API and for development until we start working on API 2.0. But since we want to get the API out faster than we can have the implementation out, I expect to keep merging from time to time. Hopefully in smaller steps than this one.

Reply:
You shouldn't have to re-do the merge. I was thinking you could do a second merge to pick up the "Remove obsolete test case "ECDH-only public key" commit.